tests/acquisition/covidcast/test_csv_importer.py

```diff
@@ -203,40 +203,30 @@ def make_row(
       (make_row(missing_val='missing_val'), 'missing_val'),
       (make_row(missing_se='missing_val'), 'missing_se'),
       (make_row(missing_sample_size='missing_val'), 'missing_sample_size'),
-      (make_row(val='1.2', missing_val=Nans.OTHER), 'missing_val'),
-      (make_row(se='1.2', missing_se=Nans.OTHER), 'missing_se'),
-      (make_row(sample_size='1.2', missing_sample_size=Nans.OTHER), 'missing_sample_size')
+      (make_row(val='1.2', missing_val=str(float(Nans.OTHER))), 'missing_val'),
+      (make_row(se='1.2', missing_se=str(float(Nans.OTHER))), 'missing_se'),
+      (make_row(sample_size='1.2', missing_sample_size=str(float(Nans.OTHER))), 'missing_sample_size'),
```

Comment:
I think this will work, but I was deleting those test cases because they were feeding `missing_*` with int values into `extract_and_check_row`, which never happens since the code converts everything to str...

(Minor vent: grrrrr this code is so unclear about the data types it's dealing with and therefore has to be written in such a defensive way, I really want to blow it up and impose stricter data types on it, but even though that will likely work for covidcast, I don't know about the other data sources delphi-epidata handles >:^0!!!).